src/ip/Intercept.cc

```diff
@@ -387,26 +385,22 @@ Ip::Intercept::Lookup(const Comm::ConnectionPointer &newConn, const Comm::Connec
 
     debugs(89, 5, "address BEGIN: me/client= " << newConn->local << ", destination/me= " << newConn->remote);
 
-    newConn->flags |= (listenConn->flags & (COMM_TRANSPARENT|COMM_INTERCEPTION));
```

Comment:
The official code did this for both `COMM_TRANSPARENT` and `COMM_INTERCEPTION` cases. Was that code duplicated? In other words, does the PR code still have similar flags copying code somewhere?